src/Pages/Main/Components/BestItem/BestItem.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import BestProductItem from './BestProductItem';
+import './BestItem.scss';
+
+class BestItem extends Component {
+  state = {
+    productData: [],
+  };
+
+  componentDidMount() {
+    fetch('/data/categoryData.json')
+      .then(result => result.json())
+      .then(productData => {
+        productData.products_list.map((data, i) => {
+          return (data['bestCount'] = i + 1);
+        });
+        this.setState({ productData: productData.products_list });
```

Comment:
- map 함수는 배열을 순회한 뒤 그 리턴값으로 새로운 배열을 리턴합니다. 만약 리턴값이 필요 없고 배열을 순회하기만 할 것이라면 forEach가 더욱 적합해보입니다.
```suggestion
      .then(({ products_list }) => {
        products_list.forEach(({ bestCount }, i) => bestCount = i + 1);
        
        this.setState({ productData: productData.products_list });
```
- map을 사용할 거라면 다음과 같이 사용하는 것이 자연스러워보입니다.
- 
```suggestion
      .then(({ products_list }) => {
        const productData = products_list.map((data, i) => ({ ...data, bestCount: i + 1 }));
        
        this.setState({ productData });
```